src/_guides/language/language-tour.md

```diff
@@ -2989,6 +2989,12 @@ Vector class, you might define a `+` method to add two vectors.
 `–`  | `%`  | `>>`
 {:.table}
 
+<div class="alert alert-info" markdown="1">
+**Note** You may have noticed that `!=` is not an overridable operator. 
```

Comment:
* Note -> Note:
* , the -> ; the
* Indent (two spaces) everything between the div tags.